### ISSUE TITLE
Send Message

### ISSUE THREAD

**Issue by iurimatias** [status-im/nim-status#196]
requires #195 , to start the message should be sent to all channels

**Comment by michaelsbradleyjr** [comment]
Sending a message currently works, but the topic can't be changed after starting the program, not can other topics be added removed. So all messages are sent and received re: that one topic.

The default topic is `/toy-chat/2/huilong/proto`, and it be changed by passing a cli option:
```
build/chat --waku-content-topic:"/toy-chat/2/example/proto"
```

There ended being supporting changes for this issue included in the finalized PR for the `/login` and `/logout` commands, which will be merged shortly.

Once I've added the ability to join multiple topics (#195), this issue will be unblocked and can be wrapped up quickly.

**Comment by michaelsbradleyjr** [comment]
Closed by #210.
